meta.schema.json

```diff
@@ -125,6 +125,12 @@
     "title": {
       "type": "string",
       "minLength": 3
+    },
+    "dcx:type": {
```

Comment:
Suggest we call this "dcx:typeRestriction". It's not a media type in the same sense as dc:format, or the "type" format in the DCX manifest, but simply calling it "dcx:type" suggest that it is.

Reply:
Agree, I've made this change now.